docs/README.md

```diff
@@ -151,8 +151,8 @@ not to be displayed in the documentation, you can do so by specifying which meth
     - save_vocabulary
 ```
 
-If you just want to add a method that is not documented (for instance magic method like `__call__` are not documented
-byt default) you can put the list of methods to add in a list that contains `all`:
+If you just want to add a method that is not documented (for instance magic method like `__call__` is not documented
+by default) you can add the list of methods to add in a list that contains `all`:
```

Comment:
I think it sounded better before to avoid using two `add`'s in the same sentence

```suggestion
by default) you can put the list of methods to add in a list that contains `all`:
```